var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -32,6 +32,7 @@ class SuperluDist(Package):
     homepage = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/"
     url = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.1.tar.gz"
 
+    version('xsdk-0.1.0', git='https://github.com/xiaoyeli/superlu_dist', commit='0b5369f')
```

Comment:
point being is that you can't build `petsc@3.7:` with `superlu-dist@xsdk-0.1.0` because petsc `depends_on('superlu-dist@5.0.0:', when='@3.7:+superlu-dist+mpi')`